### ISSUE TITLE
Audit remaining src-to-experiments dependencies after runtime defaults migration

### ISSUE THREAD

**Issue by t22cs011** [t22cs011/braindecodetest#272]
## Context

Follow-up to:
- #264 legacy-but-active `experiments` dependency audit
- #266 behavior-preservation tests
- #268 metadata/cache contract alias migration
- #270 read-only class-label/default runtime defaults migration
- PR #271: migrated read-only runtime defaults to `src` aliases and closed #270

Current architectural direction:
- `src/` is the current runtime source of truth.
- `experiments/` should become thin compatibility wrappers / legacy entrypoints / one-off scripts.
- Structural debt is being removed before external EEG pretraining implementation.
- Silent behavior change is forbidden.
- Reproducibility preservation > clean architecture.
- One issue = one PR.

PR #271 removed the low-risk `experiments.config` imports from:
- `src/experiment/fold_postprocess.py`
- `src/experiment/runtime_spec.py`

Before migrating broader surfaces such as `experiments.models`, `experiments.augmentation`, `experiments.training_loop`, or plotting, we need an updated post-#271 dependency map and risk classification.

## Goal

Produce a current, post-#271 audit of all remaining `src -> experiments` imports and classify which remaining surfaces are safe to migrate next.

This issue is intentionally audit-first. Do not migrate model registry, augmentation, training loop, or plotting yet.

## Scope

Allowed:
- Inspect latest `master` after PR #271.
- Run grep/import analysis for all `src -> experiments` dependencies.
- Produce a focused Markdown audit document under `docs/audits/` or update the existing legacy dependency audit document if one already exists.
- Add a small regression test only if it documents the current boundary without changing behavior.
- Identify the next recommended migration slice.

Not allowed:
- Do not move `experiments.models` / `MODEL_SPECS` yet.
- Do not move augmentation code.
- Do not move training loop code.
- Do not move plotting code.
- Do not change preprocessing.
- Do not change selected channels.
- Do not change fold setup / standardization.
- Do not change cropped decoding.
- Do not change W&B behavior or output layout.
- Do not implement pretraining.
- Do not run full training.

## Required audit commands

Run and include outputs/summaries in the audit:

```bash
git grep -n "from experiments\|import experiments" src | cat

git grep -n "MODEL_SPECS\|experiments.models\|experiments.augmentation\|experiments.training_loop\|experiments.plotting" src experiments tests | cat

git grep -n "experiments.config" src experiments tests | cat
```

Also inspect import direction manually to identify whether each remaining dependency is:

1. read-only constant / alias
2. model registry / construction
3. runtime orchestration
4. augmentation behavior
5. plotting/reporting only
6. legacy compatibility wrapper
7. one-off script or dead surface

## Audit document requirements

The audit must include:

1. Summary table

Columns:
- file
- imported `experiments.*` surface
- why it is imported
- runtime-critical? yes/no
- behavior-risk level: low / medium / high
- recommended action
- recommended issue number placeholder, e.g. `next`, `later`, `do not migrate now`

2. Post-#271 status

Explicitly state that the following are already migrated and should not be reopened unless tests fail:
- BCI2020 metadata aliases
- legacy runtime cache contract aliases
- class-label/default runtime config imports from #270/#271

3. Next migration recommendation

Recommend exactly one next migration slice.

The recommendation must be conservative. Candidate options may include:
- add behavior tests for `MODEL_SPECS` before migration
- migrate a read-only model registry alias only if it can be proven behavior-preserving
- defer augmentation/training_loop/plotting because they are higher risk

4. Non-goals respected

State explicitly that this PR did not migrate models, augmentation, training loop, plotting, preprocessing, selected channels, W&B/output layout, or pretraining.

## Required checks

At minimum:

```bash
python -m compileall src experiments tests
python -m pytest tests/behavior -q
```

If the environment cannot run these, document why and rely on CI.

## PR requirements

Open a small PR.

PR body must include:
- Summary of audit/update
- Exact commands run and results
- `Runtime behavior changed? No`
- `Pretraining implementation? No`
- Recommended next issue
- `Closes #272` once the actual issue number is known

## Acceptance criteria

- A post-#271 `src -> experiments` dependency audit exists in the repo.
- Remaining active dependency surfaces are classified by risk.
- The next migration slice is clearly identified.
- No runtime behavior changes are introduced.
- CI passes.
